controllers/nicclusterpolicy_controller.go

```diff
@@ -115,6 +122,39 @@ func (r *NicClusterPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	return ctrl.Result{}, nil
 }
 
+func (r *NicClusterPolicyReconciler) updateNodeLabels(cr *mellanoxv1alpha1.NicClusterPolicy,
+	infoCatalog state.InfoCatalog) error {
+	// TODO: support case if MOFED is not configured
+	nodeInfo := infoCatalog.GetNodeInfoProvider()
+	if nodeInfo == nil {
+		return errors.New("catalog does not provide node information")
+	}
+	if cr.Spec.OFEDDriver != nil {
+		attrs := nodeInfo.GetNodesAttributes(
+			nodeinfo.NewNodeLabelFilterBuilder().WithLabel(nodeinfo.NodeLabelMlnxNIC, "true").Build())
+		var pods corev1.PodList
+		appLabel := "mofed-" + attrs[0].Attributes[nodeinfo.AttrTypeOSName] + attrs[0].Attributes[nodeinfo.AttrTypeOSVer]
+		_ = r.Client.List(context.TODO(), &pods, client.MatchingLabels{"app": appLabel})
+		for _, pod := range pods.Items {
+			labelValue := "true"
+			if pod.Status.Phase != "Running" {
```

Comment:
Will a pod be in running state when the driver container is building or during driver load ?

Reply:
Good catch, we need t check ContainerStatuses